### DISCUSSION TITLE
Disable ComboboxEditor in TemplatedDialog

### DISCUSSION THREAD

**Discussion by dominhnhien1997** [serenity-is/Serenity#7238] [Help]
I want to set readonly a combox editor, I used EditorUtils.setReadonly method and passed it as true. But then I found that the controls can still change the value. So how should I do?. Please help me
Serenity version : 8.4.8 

`EditorUtils.setReadonly(myElement, true)`

**Comment by LSNicholls** [comment]
@dominhnhien1997 , it may be a question of where you are putting that code.

I just tried this in the UserDialog; it worked when I put it in afterLoadEntity but not in the constructor.
```javascript
  EditorUtils.setReadonly(this.form.TwoFactorAuth.domNode, true);     
```

of course, if this is not conditional, you can always set the item readonly in the form instead -- just want to make sure you know this:
``` c#
[ReadOnly(true)]
public TwoFactorAuthType TwoFactorAuth { get; set; }
    
```
